### PR TITLE
adding 2015 proposals page for a start to 2016 proposals page

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: pages/proposals.jsx [hunk 1]
```diff
@@ -2,21 +2,553 @@ var React = require('react');
...
+        </div>
+        {bottomLabel}
+        {errorElement}
+      </div>
+    );
+  }
+});
 
 var Proposals = React.createClass({
+  submit: function(fields) {},
```
Comment (ScottDowne) [inline pages/proposals.jsx]: I need t flesh out this function to grab the data from the fields based on field names passed in. Then send that to the /add-session endpoint and ensure it enters the google sheet I'm setting up now.

File: pages/proposals.jsx [hunk 2]
```diff
@@ -2,21 +2,553 @@ var React = require('react');
...
+  /*onSubmit: function() {
```
Comment (ScottDowne) [inline pages/proposals.jsx]: I've been keeping this as a reference, but it's last years, and I'll remove this comment once I finish the submit function directly above.

**Comment by ScottDowne** [comment]
@mmmavis r?

**Comment by mmmavis** [comment]
sorry missed the ping. reviewing this PR now.

File: app.js [hunk 3]
```diff
@@ -30,6 +31,80 @@ app.configure(function() {
...
+      "entry.1376483821": descAnotherLang,
+      "entry.928325971": descTravel,
+      "entry.2085186584": descOtherSpace
+    }
+  }, function(err) {
+    if (err) {
+      res.status(500).send({error: err});
+    } else {
+      hatchet.send("mozfest_session_proposal", {
+        email: email
```
Comment (mmmavis) [inline app.js]: What does this do? Sent a transactional email to the proposal submitter?

File: pages/proposals.jsx [hunk 4]
```diff
@@ -2,21 +2,628 @@ var React = require('react');
...
+        <InputCombo errorMessage="Email is required." for="email" type="text" name="email">
+          Email *
+        </InputCombo>
+        <InputCombo for="organization" type="text" name="organization">
+          <div>Organisation</div>
+          <div>Please leave blank if you don't want it published.</div>
+        </InputCombo>
+        <InputCombo for="otherFacilitators" type="text" name="other-facilitators">
+          <div>Other facilitators</div>
+          <div>One per line, format is Name[single space]Twitter handle [single space]email (e.g. John Doe @j.doe j.doe@example.com)</div>
```
Comment (mmmavis) [inline pages/proposals.jsx]: @ScottDowne could we make the values commas-separated (instead of using single spaces)?

File: pages/proposals.jsx [hunk 5]
```diff
@@ -2,21 +2,628 @@ var React = require('react');
...
+      });
+    }
+  },
+  renderStepOne: function() {
+    return (
+      <div>
+        <InputCombo errorMessage="First name is required." for="firstName" type="text" name="first-name">
+          First Name *
+        </InputCombo>
+        <InputCombo errorMessage="Surname name is required." for="surname" type="text" name="surname">
```
Comment (mmmavis) [inline pages/proposals.jsx]: extra word "name" in `errorMessage` 😉

File: pages/proposals.jsx (hunk 4 above)
Comment (mmmavis) [inline pages/proposals.jsx]: `One per line, format is Name[comma] Twitter handle[comma] email (e.g. John Doe, @j.doe, j.doe@example.com)`

File: pages/proposals.jsx [hunk 6]
```diff
@@ -2,21 +2,628 @@ var React = require('react');
...
+
+        <InputCombo wordcount="150" errorMessage="Field is required." for="desc-allow" name="desc-allow" type="textarea">
+          Describe what your session or activity will allow people to make, learn or do in 150 words or less. *
+        </InputCombo>
+
+        <InputCombo wordcount="150" errorMessage="Field is required." for="desc-working" name="desc-working" type="textarea">
+          Describe how you see that working in 150 words or less. *
+        </InputCombo>
+
+        <InputCombo errorMessage="Field is required." for="participants" name="participants" type="text">
```
Comment (mmmavis) [inline pages/proposals.jsx]: Should this be a `textarea` as well?

File: pages/proposals.jsx [hunk 7]
```diff
@@ -2,21 +2,628 @@ var React = require('react');
...
+        </InputCombo>
+
+        <label>MozFest offers limited places for travel sponsorship (stipends) covering flights and accommodation over the festival weekend. These places are offered to those who need support traveling to the event and would have no other means to attend through work or personal covering of costs. We offer these stipends to encourage diversity in facilitators. *</label>
+        <InputCombo className="radio-input" name="travel-radio" for="travel-only" type="radio">
+          I can only attend MozFest if I receive a travel stipend covering my travel and accommodation
+        </InputCombo>
+        <InputCombo className="radio-input" name="travel-radio" for="travel-none" type="radio">
+          I do not require a travel stipend as I, or my work, can cover my costs
+        </InputCombo>
+        <div id="travel-radioError" className="error-message">Travel stipend is required.</div>
```
Comment (mmmavis) [inline pages/proposals.jsx]: We should probably tweak the error message. `Travel stipend is required.` makes it sound like the proposal will be accepted *only if* Mozilla is paying for the trip.

File: pages/proposals.jsx [hunk 8]
```diff
@@ -2,21 +2,628 @@ var React = require('react');
...
+
+        <label>If your session is not accepted in your preferred Space is there another space you would like your session to be considered within? Read details on each Space  <a href="/spaces-and-sessions">here</a>.</label>
```
Comment (mmmavis) [inline pages/proposals.jsx]: another space -> another Space

**Comment by mmmavis** [comment]
🎉🎉🎉🎉🎉 R+ thanks @ScottDowne 
( I didn't test the Spanish, French and Arabic forms though 😉 )

**Pull request merged**